README.md

```diff
@@ -1,6 +1,10 @@
+## Deprecated
```

Comment:
Minor: update this to a single # so it displays as an h1. The heading below is also an h1 so updating this would follow the pattern of a consistent hierarchy.